apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -121,7 +122,8 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
       state.dutchAuction.auctionInfos = [];
     });
 
-    for await (const response of viewClient.auctions({})) {
+    /** @todo: Sort by... something? */
```

Comment:
what's the motivation here?

Reply:
Eventually, I'll probably want to sort by e.g., most recent auction descending, or something.